File: Easy/age_distribution.py
data = ['0', '19']

#import sys
#test_cases = open(sys.argv[1], 'r')
#data = test_cases.readlines()
for test in data:
	test = test.rstrip()
	age = int(test)
	if ((age >= 0) and (age <= 2)):
		print "Still in Mama's arms"
	elif ((age == 3) or (age == 4)):
		print "Preschool Maniac"
	elif ((age >= 5) and (age <= 11)):
		print "Elementary school"
	elif ((age >= 12) and (age <= 14)):
		print "Middle school"
	elif ((age >= 15) and (age <= 18)):
		print "High school"
	elif ((age >= 19) and (age <= 22)):
		print "College"
	elif ((age >= 23) and (age <= 65)):
		print "Working for the man"
	elif ((age >= 66) and (age <= 100)):
		print "The Golden Years"
	else:
		print "This program is for humans"

#test_cases.close()

File: Easy/age_distribution.r
test.cases <- c("0", "19")

age_desc <- function(age){
	if ((age >= 0) & (age <= 2)){
		"Still in Mama's arms"
	}
	else if((age == 3) | (age == 4)) {
		"Preschool Maniac"
	}
	else if((age >= 5) & (age <= 11)) {
		"Elementary school"
	}
	else if((age >= 12) & (age <= 14)) {
		"Middle school"
	}
	else if((age >= 15) & (age <= 18)) {
		"High school"
	}
	else if((age >= 19) & (age <= 22)) {
		"College"
	}
	else if((age >= 23) & (age <= 65)) {
		"Working for the man"
	}
	else if((age >= 66) & (age <= 100)) {
		"The Golden Years"
	}
	else {
		"This program is for humans"
	}
}

#args <- commandArgs(trailingOnly=TRUE)
#test.cases <- strsplit(readLines(args[[1]], warn=FALSE), '\n')
cat(sapply(test.cases, function(test) {
	if (length(test) > 0) {
		AGE <- as.integer(test)
		age_desc(AGE)
	}
}), sep = "\n")